distribution/build.gradle

```diff
@@ -167,6 +167,7 @@ buildDefaultLog4jConfig.doLast(writeLog4jProperties)
 
 // copy log4j2.properties from modules that have it
 void copyLog4jProperties(Task buildTask, Project module) {
+  buildTask.dependsOn { module.bundlePlugin }
```

Comment:
Is there a reason we are deferring this instead of just doing `buildTask.dependsOn module.bundlePlugin`?

Reply:
The module might not have been configured yet, so there might not yet be a bundlePlugin task to refer to.

Reply:
Then we can use a string here `dependsOn "$module:bundlPlugin"`

Reply:
In general the string approach also produces a slightly nicer error. Something along the lines of "can't find a task with path :foo" rather than a more generic "could not evaluate property 'bundlePlugin' on foo".